src/proposals/mips/mip-m18/mip-m18d.sol

```diff
@@ -271,6 +275,66 @@ contract mipm18d is HybridProposal, MultichainGovernorDeploy {
             "Set the pending admin of mETHwh to the Multichain Governor",
             true
         );
+
+        delete cTokenConfigurations[block.chainid]; /// wipe existing mToken Configs.sol
+        delete emissions[block.chainid]; /// wipe existing reward loaded in Configs.sol
+
+        {
+            string
+                memory mtokensPath = "./src/proposals/mips/mip-m18/mip-m23.json";
```

Comment:
It seems strange to me that the mip-m23 JSON file is located inside the mip m18 folder. Do we need to run both together?

Reply:
could you also change the folder and file names?